plugins/org.jboss.tools.openshift.ui/src/org/jboss/tools/openshift/internal/ui/explorer/OpenShiftExplorerContentProvider.java

```diff
@@ -45,17 +45,12 @@
 public class OpenShiftExplorerContentProvider implements ITreeContentProvider {
 	private static final List<String> TERMINATED_STATUS = Arrays.asList("Complete", "Failed", "Error", "Cancelled");
 
-	private OpenshiftUIModel model;
+	private OpenshiftUIModel model = OpenshiftUIModel.getInstance();
 	private IElementListener listener;
 	private StructuredViewer viewer;
 	private Map<Object, BaseExplorerContentProvider.LoadingStub> stubs = new HashMap<Object, BaseExplorerContentProvider.LoadingStub>();
 
 	public OpenShiftExplorerContentProvider() {
-		this(new OpenshiftUIModel());
-	}
-
-	public OpenShiftExplorerContentProvider(OpenshiftUIModel model) {
```

Comment:
keeping that constructor makes is more testable as you could inject a mock OpenshiftUIModel